addons/spreadsheet/static/src/commands.json

```diff
@@ -333,24 +263,17 @@
           "y": 10
       },
       "definition": {
-          "metaData": {
-              "groupBy": [
-                  "stage_id",
-                  "country_id"
-              ],
-              "measure": "__count",
-              "order": "ASC",
-              "resModel": "crm.lead"
-          },
-          "searchParams": {
-              "comparison": null,
-              "context": {},
-              "domain": [],
-              "groupBy": [
-                  "stage_id",
-                  "country_id"
-              ],
-              "orderBy": []
+          "definition": {
```

Comment:
`definition.definition` ? :eyes: 
I'd rather avoid mixing standard chart related attributes from "odoo datasource" things

Reply:
but I don't think of any good alternative name for the data source definition vs chart definition

Reply:
`odooDefinition` ? At least it's clear that it's odoo related stuff.